codecov_cli/services/upload/upload_sender.py

```diff
@@ -176,9 +185,15 @@ def get_url_and_possibly_update_data(
         encoded_slug,
         commit_sha,
         report_code,
+        combined_upload=False,
     ):
         if report_type == "coverage":
-            url = f"{upload_url}/upload/{git_service}/{encoded_slug}/commits/{commit_sha}/reports/{report_code}/uploads"
+            if combined_upload:
+                url = (
+                    f"{upload_url}/upload/{git_service}/{encoded_slug}/combined-upload"
+                )
+            else:
+                url = f"{upload_url}/upload/{git_service}/{encoded_slug}/commits/{commit_sha}/reports/{report_code}/uploads"
```

Comment:
the formatting looks a bit weird. how about this:
```suggestion
            root_url = f"{upload_url}/upload/{git_service}/{encoded_slug}/"
            if combined_upload:
                url = f"{root_url}/combined-upload"
            else:
                url = f"{root_url}/commits/{commit_sha}/reports/{report_code}/uploads"
```

Reply:
This looks great, thanks for the suggestion!